### DISCUSSION TITLE
Ability to name a loop and break it with break

### DISCUSSION THREAD

**Discussion by MaKeSter333** [dotnet/csharplang#7278] [Language Ideas]
`
while (true): my_while
{
    while (true)
    {
        while (true)
        {
            break my_while;
        }
    }
}
`
This will allow you to conveniently exit nested loops

**Comment by svick** [comment]
This is effectively a duplicate of https://github.com/dotnet/csharplang/discussions/6634, though the specific syntax a matter of discussion.
